### PR TITLE
fix(rum-core): measure fid correctly for experience metric

### LINKED ISSUE #898: FID experience metric is measured incorrectly
FID should measure the duration between Input start and Processing start, Right now we are setting it to start value of FID span which is relative to time origin which is incorrect.

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ fixes #898 
+ Also handles CLS values to be fixed with 3 decimals (Used in all core-vitals reports)

### CODE REVIEW

**Review by jahtalab — COMMENTED** [review]
Thanks @vigneshshanmugam , It seems that tests are failing, I suspect it's because of the wrong data type (see comments)

File: packages/rum-core/src/performance-monitoring/transaction-service.js [hunk 1]
```diff
@@ -287,7 +287,7 @@ class TransactionService {
             }
 
             if (isPerfTypeSupported(LAYOUT_SHIFT)) {
-              tr.experience.cls = cls
+              tr.experience.cls = cls.toFixed(3)
```
[the remark this answers is not in the document]
Comment (jahtalab) [reply packages/rum-core/src/performance-monitoring/transaction-service.js:290]:
```suggestion
              tr.experience.cls = cls
```

**Comment by jahtalab** [comment]
@vigneshshanmugam, I've reverted the fixed decimals for cls, let's do that in the UI if necessary.

**Comment by vigneshshanmugam** [comment]
@jahtalab I am okay with doing formatting on UI as well, It just felt right doing it from RUM side as we never send long values for other fields. it seems odd to do it only for CLS. We can do it in separate PR as well if you have any other concerns.

File: packages/rum-core/src/performance-monitoring/transaction-service.js (hunk 1 above)
Comment (vigneshshanmugam) [inline packages/rum-core/src/performance-monitoring/transaction-service.js]:
```suggestion
              tr.experience.cls = parseFloat(cls.toFixed(3))
```

**Review by jahtalab — APPROVED** [review]
As we discussed, I think we should truncate cls but let's decide that separately.

**Pull request merged**